src/test/java/com/exasol/releasedroid/usecases/release/ReleaseInteractorTest.java

```diff
@@ -57,16 +60,27 @@ void setup() {
     }
 
     private List<Report> release(final PlatformName... platforms) {
-        return release(asList(platforms), emptySet());
+        return release(false, asList(platforms), emptySet());
     }
 
     private List<Report> release(final List<PlatformName> platforms, final Set<PlatformName> skipValidationOn) {
-        final Map<PlatformName, ReleaseMaker> releaseMarkers = Map.of( //
+        return release(true, platforms, skipValidationOn);
+    }
+
+    private List<Report> release(final boolean mockEstimation, final List<PlatformName> platforms,
+            final Set<PlatformName> skipValidationOn) {
+        final Map<PlatformName, ReleaseMaker> releaseMakers = Map.of( //
                 PlatformName.GITHUB, this.githubReleaseMakerMock, //
                 PlatformName.JIRA, this.jiraReleaseMakerMock, //
                 PlatformName.MAVEN, this.mavenReleaseMakerMock, //
                 PlatformName.COMMUNITY, this.communityReleaseMakerMock);
-        final ReleaseInteractor releaseInteractor = new ReleaseInteractor(this.validateUseCaseMock, releaseMarkers,
+        if (mockEstimation) {
+            for (final PlatformName p : platforms) {
+                final ReleaseMaker maker = releaseMakers.get(p);
+                when(maker.estimateDuration(any())).thenReturn(Estimation.of(Duration.ofSeconds(1)));
```

Comment:
Extract this into a method that configures each mock separetly, this would make it easier to understand.

Reply:
👍